src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -322,6 +322,8 @@
             importProgram: mapActionByName("surveyAndProgram", "importProgram"),
             importANC: mapActionByName("surveyAndProgram", "importANC"),
             async importDataset() {
+                this.stopPolling();
```

Comment:
I don't understand what this stop/start combination is achieving? To me it looks like polling should be started here conditional on there not being a release selected, and similarly the polling that is started on mount should be conditional on there not being a release selected.

Reply:
This ensures polling does not continue when switching from Latest to release dataset.